examples/plot_firing_pattern.py

```diff
@@ -15,8 +15,7 @@
 # Let us import hnn_core
```

Comment:
should we rename the example to say `plot_somatic_voltages.py` or something like that?

Reply:
I say keep the name, we can add a plot to this file for currents in the next PR.